pkg/golinters/gci/internal/analyzer.go

```diff
@@ -108,7 +108,7 @@ func runAnalysis(pass *analysis.Pass) (any, error) {
 
 		pass.Report(analysis.Diagnostic{
 			Pos:            fix.TextEdits[0].Pos,
-			Message:        "Invalid import order",
+			Message:        "File is not properly formatted",
```

Comment:
Perhaps we should leave "Invalid import order" message, as this suggests that something is wrong with imports.

E.g., `File is not properly formatted: invalid import order`.

Reply:
The similar for `goimports`.

Reply:
The formatters format all the code, not only imports.
So a message only related to imports is a bad idea: if a report points a line in the middle or the end of the file, the message will be confusing.

Reply:
for the what reason "imports formatter" formats not imports only, but all code oO?

Reply:
"imports formatters" don't exist: `goimports` is a variant of `gofmt` with an option to sort imports.

`goimports` has 2 modes "format-only" (gofmt) or "format and sort imports":
```console
$ goimports -h                              
usage: goimports [flags] [path ...]
...
  -format-only
        if true, don't fix imports and only format. In this mode, goimports is effectively gofmt, with the addition that imports are grouped into sections.
...
```

Reply:
Also `gci` is like `goimports` and always formats the code because it's needed.

Reply:
Thank you for explanation 👍 

I guess (from my experience) the root cause in common usage of `go/format.Format` inside linters.

Maybe to increase transparency and warn something like
```
File is not properly formatted (please, check code style including imports order) 
```
?

Reply:
> I guess (from my experience) the root cause in common usage of go/format.Format inside linters.

This is not the root cause of why formatters format the code :wink: 
There are some usages of the package `go/format` but not only that, it's a bit more subtle.

> Maybe to increase transparency and warn something like:
> File is not properly formatted (please, check code style including imports order)
> ?

This is not a log but a report message: the message should be precise, short, and without parenthesis if possible.

Also, the report style has changed: since my previous PR about suggested fixes, the report positions are more precise than before, and there are already fewer ambiguities.